dts/msm8953-xiaomi-vince.dts

```diff
@@ -9,18 +9,27 @@
 	qcom,msm-id = <293 0x0>;
 	qcom,board-id= <0x1000b 0x8>;
 
-	model = "Xiaomi Redmi Note 5/5+ (vince)";
+	model = "Xiaomi Redmi 5 Plus";
 	compatible = "xiaomi,vince", "qcom,msm8953", "lk2nd,device";
-	lk2nd,match-bootloader = "*DAISY1.0*";
 	lk2nd,pstore = <0x9ff00000 0x100000>;
+	lk2nd,match-bootloader = "*DAISY1.0*";
```

Comment:
xiaomi-daisy uses the same bootloader string. I didn´t specify it, but it also is "DAISY1.0".
https://github.com/msm8953-mainline/lk2nd/blob/45c6b17e92a46ecaee4cc9bf8eb5ae1ec8f6a9cc/dts/msm8953-xiaomi-daisy.dts#L14
Maybe we should find another way to differenciate both devices.

Reply:
It is different board ids for daisy and vince, so this should be enough, because the bootloader recognizes the device correctly. Added it just because it's present in all others dts. Maybe let's just delete it?

Reply:
Ah right. Then delete it if you want, i don't think it does any harm though.